lib/cib/Makefile.am

```diff
@@ -21,7 +21,7 @@ libcib_la_SOURCES	+= cib_remote.c
 libcib_la_SOURCES	+= cib_utils.c
 
 libcib_la_LDFLAGS	= -version-info 33:0:6
-libcib_la_CPPFLAGS	= -I$(top_srcdir) $(AM_CPPFLAGS)
+libcib_la_CPPFLAGS 	= $(AM_CPPFLAGS)
```

Comment:
We don't need target_CPPFLAGS unless we need to add something to AM_CPPFLAGS (but that can go in the next PR)

Reply:
I thought that was the case too, but there was some weirdness going on with failures in CI.